src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,25 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.List;
+
+
+/**
+ * Lists all persons in the address book to the user.
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Displays all persons in the address book as a list sorted by their name"
+            + ", in alphabetical order.\n"
```

Comment:
According to your implementation:
https://github.com/nusCS2113-AY1819S2/addressbook-level2/blob/7942d0695d9c32e501ba9fda7df80a080db31127/src/seedu/addressbook/data/person/Name.java#L63-L66

You are not actually sorting the list in **alphabetical order**, but rather in **lexicographical order**.

Refer to [this relevant Quora question](https://www.quora.com/Whats-the-difference-between-lexicographical-and-alphabetical-order) for more information.